apps/web/src/app/user-locale-setter.tsx

```diff
@@ -0,0 +1,13 @@
+'use client';
+
+import { useEffect } from 'react';
+import { setCookie } from '@/app/actions';
+
+export default function UserLocaleSetter(): React.ReactNode {
+  useEffect(() => {
+    void setCookie('USER_LOCALE', navigator.language);
```

Comment:
```suggestion
    Cookies.set('USER_LOCALE', navigator.language);
```
I am getting an error in console with the action
`Uncaught (in promise) Error: Only plain objects, and a few built-ins, can be passed to Client Components from Server Components. Classes or null prototypes are not supported.`

The same is happening in `color-scheme-picker.tsx`